static/styles/zulip.css

```diff
@@ -295,25 +296,14 @@ p.n-margin {
 
 .app {
     width: 100%;
-    height: 100%;
-    overflow-y: scroll;
     z-index: 99;
     -webkit-overflow-scrolling: touch;
```

Comment:
We can remove this `-webkit-overflow-scrolling` (it should already be the default on the root scrollbar).